src/logic/getSports.js

```diff
@@ -0,0 +1,23 @@
+function getSports(age, weight, goals) {
+    let sports = ['run', 'bike', 'swim', 'row', 'walk', 'yoga', 'weights']
+    if (age > 60) {
```

Comment:
Can you do everything in one go? It would be a bit simpler to read:

```
sports.filter(sport => (
    filterWeights(sport) ||
    filterGoals(sport)
    ...
))
```